pyproject.toml

```diff
@@ -12,46 +12,43 @@ dependencies = [
     "python-magic~=0.4.27",
     "python-dotenv==1.0.0",
     # Adapter changes
-    "llama-index==0.10.38",
-    # Temporary hack to get out of llama-index issues
-    # To be removed once llama-index version is upgraded
-    "llama-index-core==0.10.56",
```

Comment:
This is removed as the code required to stay compatible with latest llama-index-core 0.10.58 is done in token_counter.py below.